Robus/src/robus.c

```diff
@@ -33,9 +33,17 @@ typedef struct __attribute__((__packed__))
     };
 } node_bootstrap_t;
 
+static network_lock_t node_connected = {
+    .state       = NETWORK_LINK_DOWN,
+    .timeout_run = false,
+    .timeout     = 0};
+
+#define NETWORK_TIMEOUT 100
```

Comment:
What is this timeout thing? we should add a small comment here. Is something that should be on node_config or product_config?